src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -160,60 +160,64 @@ private AdsDataProvider() {
 
   @DataProvider
   public static Object[][] skin() {
-    final String wikiName = "project43";
-    final String pageName = "SyntheticTests/Skin";
+    final String noSkinWikiName = "adtest";
```

Comment:
IMO My point here -- to have one approach for whole file.
(I don't see the value just doing it for one data provider method)